aiohttp/__init__.py

```diff
@@ -14,11 +14,11 @@
     ClientOSError,
     ClientPayloadError,
     ClientProxyConnectionError,
-    ClientResponse,
     ClientRequest,
+    ClientResponse,
```

Comment:
@asvetlov could you please just commit style changes into master so that it'd not show up here making it harder to review.

Reply:
Yes, sure.
Thanks for the recall.
I don't wont to see this mess in the PR :)